### PR TITLE
READMEs for arctic dem add-on

### PR DESCRIPTION
**Pull request by trey-stafford**
## Description

This PR is to aid in the review of READMEs for the arctic dem addon packages.

Packages live here: https://data.pgc.umn.edu/gis/packages/qgreenland/packages/addons/

## Checklist

If an item on this list is done _or not needed_, check it with `[x]` or click the
checkbox.

- [ ] The PR description links to issues that it resolves with `closes #{issue_number}`
- [ ] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [ ] Environment lockfile updated if needed (`conda-lock`)
- [ ] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build)`)
- [ ] CHANGELOG.md updated (for user-facing changes)
- [ ] Documentation updated if needed
- [ ] New unit tests if needed

### CODE REVIEW

File: qgreenland/addons-readme.txt
Comment (trey-stafford) [inline qgreenland/addons-readme.txt:1]: This file contains text that can replace what's currently on our website for ArcticDEM:

> [32-meter resolution ArcticDEM](http://data.pgc.umn.edu/gis/packages/qgreenland/packages/addons/) (digital elevation model): The Polar Geospatial Center has provided a custom Greenland 32-m resolution ArcticDEM for added detail. Download this add-on package and place it in a suitable location in your filesystem. Unzip the package and then follow the directions in the 'README.txt' file included.

File: qgreenland/addons-readme.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+32-meter resolution ArcticDEM (digital elevation model): The Polar Geospatial
+Center has provided a custom Greenland 32-m resolution ArcticDEM for added
```
Comment (MattF-NSIDC) [inline qgreenland/addons-readme.txt:2]:
```suggestion
Center has provided a custom Greenland 32m resolution ArcticDEM for added
```

File: qgreenland/arctic-dem-dem-only-readme.txt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Building Overviews:
+-------------------
+
+Because this dataset is large and consists of many smaller files, performance of
+the layer after initially loading it into QGIS may be slow. We recommend that
+users build raster overviews for this data, which will increase the rendering
+speed when zooming in/out with the 32m Arctic DEM enabled.
+
+To build overviews in QGIS, navigate in the Menu Bar to `Raster -> Miscellaneous
+-> Build Overviews (Pyramids)` . Select the 32m Arctic DEM as the input layer,
```
Comment (MattF-NSIDC) [inline qgreenland/arctic-dem-dem-only-readme.txt:25]:
```suggestion
-> Build Overviews (Pyramids)`. Select the 32m Arctic DEM as the input layer,
```

File: qgreenland/arctic-dem-dem-only-readme.txt [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+* QGIS documentation on raster symbology properties:
+  https://docs.qgis.org/3.28/en/docs/user_manual/working_with_raster/raster_properties.html#symbology-properties
+
+
+-------------------
+Citing this data:
+-------------------
+
+The ArcticDEM dataset should be cited as follows
+Porter, C., Howat, I., Noh, M.J., Husby, E., Khuvis, S., Danish, E., Tomko, K.,
```
Comment (MattF-NSIDC) [inline qgreenland/arctic-dem-dem-only-readme.txt:50]:
```suggestion
The ArcticDEM dataset should be cited as follows:

Porter, C., Howat, I., Noh, M.J., Husby, E., Khuvis, S., Danish, E., Tomko, K.,
```

File: qgreenland/arctic-dem-full-package-readme.txt [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+* QGIS documentation on raster symbology properties:
+  https://docs.qgis.org/3.28/en/docs/user_manual/working_with_raster/raster_properties.html#symbology-properties
+
+
+-------------------
+Citing this data:
+-------------------
+
+The ArcticDEM dataset should be cited as follows
+Porter, C., Howat, I., Noh, M.J., Husby, E., Khuvis, S., Danish, E., Tomko, K.,
```
Comment (MattF-NSIDC) [inline qgreenland/arctic-dem-full-package-readme.txt:72]:
```suggestion
The ArcticDEM dataset should be cited as follows:

Porter, C., Howat, I., Noh, M.J., Husby, E., Khuvis, S., Danish, E., Tomko, K.,
```

File: qgreenland/arctic-dem-full-package-readme.txt [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+Building Overviews:
+-------------------
+
+Because each layer is large and consists of many smaller files, performance of
+each layer after initially loading it into QGIS may be slow. We recommend that
+users build raster overviews for each layer, which will increase the rendering
+speed when zooming in/out.
+
+To build overviews for a layer in QGIS, navigate in the **Menu Bar** to **Raster
+-> Miscellaneous -> Build Overviews (Pyramids)** . Select the layer as the input
```
Comment (MattF-NSIDC) [inline qgreenland/arctic-dem-full-package-readme.txt:47]:
```suggestion
To build overviews for a layer in QGIS, navigate in the **Menu Bar** to `Raster
-> Miscellaneous -> Build Overviews (Pyramids)`. Select the layer as the input
```

For consistency with above

File: qgreenland/arctic-dem-dem-only-readme.txt [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+-------------------
+Building Overviews:
+-------------------
+
+Because this dataset is large and consists of many smaller files, performance of
+the layer after initially loading it into QGIS may be slow. We recommend that
+users build raster overviews for this data, which will increase the rendering
+speed when zooming in/out with the 32m Arctic DEM enabled.
+
+To build overviews in QGIS, navigate in the Menu Bar to `Raster -> Miscellaneous
```
Comment (MattF-NSIDC) [inline qgreenland/arctic-dem-dem-only-readme.txt:24]:
```suggestion
To build overviews in QGIS, navigate in the **Menu Bar** to `Raster -> Miscellaneous
```

**Review by MattF-NSIDC — APPROVED** [review]

**Pull request merged**